### ISSUE TITLE
Bazel CI: rules_rust is failing on darwin

### ISSUE THREAD

**Issue by meteorcloudy** [bazelbuild/rules_rust#67]
https://ci.bazel.build/blue/organizations/jenkins/rules_rust/detail/rules_rust/800/pipeline

suspect: 2a3f859
@davidzchen Can you take a look?

```
running 1 test
test test::test_can_read_data_from_runfiles ... FAILED
failures:
---- test::test_can_read_data_from_runfiles stdout ----
thread 'test::test_can_read_data_from_runfiles' panicked at 'called `Result::unwrap()` on an `Err` value: Error { repr: Os { code: 2, message: "No such file or directory" } }', src/libcore/result.rs:860:4
note: Run with `RUST_BACKTRACE=1` for a backtrace.
failures:
test::test_can_read_data_from_runfiles
test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out
```

**Comment by acmcarther** [comment]
https://github.com/bazelbuild/rules_rust/pull/66 fixes this, waiting on review from another maintainer.

**Comment by acmcarther** [comment]
Was fixed with #66
